framework/src/constraints/MortarConstraintBase.C

```diff
@@ -187,18 +187,24 @@ MortarConstraintBase::zeroInactiveLMDofs(const std::unordered_set<const Node *>
   if (!_var)
     return;
 
+  const auto sn = _sys.number();
+  const auto vn = _var->number();
```

Comment:
What `sn` and `vn` stand for?

Reply:
Haha I can change the variable names to sys_number and var_number